.asf.yaml

```diff
@@ -61,6 +61,8 @@ github:
         # contexts are the names of checks that must pass
         contexts:
           - check
+          - lint
+          - test-mysql
```

Comment:
I think we may need to append python version to `test-mysql` as well. E.g. `test-mysql (3.7)`.

It seems GitHub checks uses the job name with matrix variables.

Reply:
ok, good call.